docs/source/usage.rst

```diff
@@ -133,6 +133,10 @@ following commands inside the directory:
   git checkout 3.5.0
   cd ..
 
+Alternative, you can download a zip (or tar.gz) file containing reveal.js from
```

Comment:
Honestly I’d prefer to keep this simpler in this location. If we really want to cover a lot of options on how to do this I think we should have a separate page. But otherwise I’d like this to be a linear flow.

Reply:
To explain why: the people who benefit most from this are going to be those who don’t quite understand why they need to do this. They’ll have the easiest time copying and pasting some commands, even if they don’t fully understand those commands. Telling someone to find a resource that they may or may not understand and then to conditionally manipulate it (`tar -xfd` or `unzip`) using tools they may or may not already know introduces unnecessary complication and uncertainty.  We should aim to be as simple as possible, which ideally implies a workflow uncluttered with as few decisions as possible.

Reply:
Fair enough, feel free to push the removal :wink: